modules/mixed-arch-creating-arm64-bootimage.adoc

```diff
@@ -0,0 +1,69 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="mixed-arch-creating-arm64-bootimage"]
+
+= Creating an `arm64` bootimage using the Azure image gallery
+ 
+Before you modify the machine set, you must manually generate the `arm64` bootimage. 
+ 
+.Prerequisites
+
+* Installed the Azure CLI (`az`).
+* Installed the OpenShift CLI (`oc`).
+* Configured your Azure portal account.
+* Created a single-architecture Azure installer-provisioned cluster with the heterogeneous installer binary. 
+
+.Procedure
+. Login to your Azure account: 
++
+[source,terminal]
+----
+az login
+----
+. Create a Microsoft storage accounts and upload the `arm64` virtual hard disk (VHD) (FIXME: To where?)
++
+[source,terminal]
+----
+az storage account create -n mystorageaccount -g MyResourceGroup -l westus --sku Standard_LRS 
+----
+. Create a storage container using the following command: 
++
+[source,terminal]
++
+----
+az storage container create -n mycontainer --account-name mystorageaccount
+----
+. Download the {op-system} VHD from the `aarch64` link:https://mirror.openshift.com/pub/openshift-v4/arm64/dependencies/rhcos/pre-release/latest-4.11/[image mirror] page.
+. Generate a shared access signature (SAS) token. Use the the token to upload the {op-system} VHD (FIXME: to where?) using the following command: 
```

Comment:
Same thing here, not sure what this upload is to?

Reply:
upload the vhd to the storage account ( the storage container in the storage account actually)